lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -514,12 +514,26 @@ function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
 function dosomething_reportback_view_entity($entity) {
   $rb_simple_user = dosomething_user_get_simple_user($rb_user = user_load($entity->uid));
   global $user;
+
   $is_owner = ($user->uid == $rb_simple_user->uid) ? 1 : 0;
   $fid = $_GET['fid'];
+
   if (!$fid) {
     // Get the most recently uploaded item.
     $fid = array_pop($entity->fids);
   }
+
+  $disable_reactions = dosomething_kudos_disable_reactions($entity->nid);
+
+  if (! $disable_reactions) {
+    $kudos = [];
+    $kudos['id'] = $entity->rbid;
```

Comment:
Is it weird that the `$kudos['id']` here is actually the reportback-item id? maybe change the name to be `$kudos['reportback_item_id']` or something that would cause less confusion, since there are ids on Kudos...